src/tacos.mvc/ClientApp/css/site.css

```
@@ -851,6 +851,178 @@ hr {
+  font-weight: 700;
+}
+
+.tacos-admin-dashboard__grid {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  gap: 1rem;
+}
+
+.tacos-admin-card {
+  --tacos-admin-card-accent: #0b3262;
+  position: relative;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)